S23/pmirukula/Conversion.java

```diff
@@ -0,0 +1,44 @@
+import java.util.*;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.*;
+import java.util.Locale;
+
+public class Conversion {
+	public void conversionlogic() 
+	{
+	}
+
+	public static void main(String[] args) throws IOException {
+		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+		System.out.println("Please enter the word");
+		String str = br.readLine();
+		System.out.println("Please enter the language you would like to translate the word");
+		String str_lang = br.readLine();
+		PrintWriter out = new PrintWriter("oceans.txt"); 

```

Comment:
It would be better to use a better name for the file to improve the readability

Reply:
Noted! will work on it.
Thanks for Suggestion!